generative/maisi/maisi_diff_unet_training_tutorial.ipynb

```diff
@@ -0,0 +1,562 @@
+{
```

Comment:
<p>Also need install xformers to run this notebook.</p><p><code></code><code></p><p>!python -c "import xformers" || pip install -q xformers --index-url <a href="https://download.pytorch.org/whl/cu121" target="_blank">https://download.pytorch.org/whl/cu121</a></p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1745/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Hi @KumoLiu, I have updated the notebook. Please proceed. Thanks.

Reply:
Closing this for now as the line command is added.